bioblend/galaxy/jobs/__init__.py

```diff
@@ -70,6 +70,48 @@ def show_job(self, job_id, full_details=False):
 
         return self._get(id=job_id, params=params)
 
+    def build_for_rerun(self, job_id):
+        """
+        Get details of a given job that can be used to rerun the corresponding tool.
+
+        :type job_id: str
+        :param job_id: job ID
+
+        :rtype: dict
+        :return: A description of the given job, with all parameters required to rerun.
+
+        """
+        url = '/'.join((self._make_url(job_id), 'build_for_rerun'))
+        return self._get(url=url)
+
+    def rerun_and_remap_job(self, job_id):
```

Comment:
Possible signature for a general `rerun_job()` method as discussed in https://github.com/galaxyproject/bioblend/pull/343/files#r516030456 :

```suggestion
    def rerun_job(self, job_id, remap=False, tool_inputs_update=None):
```